mmap+memset+fork.c

```diff
@@ -83,6 +87,12 @@ void parse_options (int argc, char *argv[]){
 			case 'n':
 				set_size = atoi (optarg);
 				break;
+			case 'u':
+				flag |= MAP_POPULATE;
+				break;
+			case 'a':
+				flag |= MAP_ANONYMOUS;
+				break;
 
```

Comment:
The top of the file still has `int flag = MAP_ANONYMOUS | MAP_POPULATE|MAP_FIXED;`  so these new optional flags will need to be removed from the default `flag` value.